mode/mllike/mllike.js

```diff
@@ -74,9 +81,18 @@ CodeMirror.defineMode('mllike', function(_config, parserConfig) {
       return 'comment';
     }
     if (/\d/.test(ch)) {
-      stream.eatWhile(/[\d]/);
-      if (stream.eat('.')) {
+      if (ch === '0' && stream.eat(/[bB]/)) {
+        stream.eatWhile(/[01]/);
+      } if (ch === '0' && stream.eat(/[xX]/)) {
+        stream.eatWhile(/[0-9a-dA-D]/)
```

Comment:
What about e and f?

Reply:
I have no idea what you meant.

if it is about the case with `1e-10` that is already covered in line 53 with the patch. Now, with the `f` case I have no idea what is that, OCaml doesn't have a 1f notation (that I know).

Please, if I am misunderstanding what you are pointing out I will appreciate it.

Now that you point that out, I notice I forgot Octal Integer Literals (`0o1234`), I am adding them now.

Reply:
The line I commented on is parsing hexadecimal literals, which use digits from 0 to f, right?

Reply:
Oh thanks mate!

I really missed that one! thanks man!

Thanks to you I got as well the long integer literals and octal 😄 

Thanks again 👍